File: bitget-golang-sdk-api/pkg/model/mix/plan/modifytpslplanreq.go
package plan

/**
 * @Author: bitget-sdk-team
 * @Date: 2022-09-30 10:46
 * @DES: modify plan request
 */
type ModifyTPSLPlanReq struct {
	/**
	 * Currency pair
	 */
	Symbol string `json:"symbol"`
	/**
	 * Deposit currency
	 */
	MarginCoin string `json:"marginCoin"`
	/**
	 * Order id
	 */
	OrderId string `json:"orderId"`
	/**
	 * Trigger price
	 */
	TriggerPrice string `json:"triggerPrice"`
	/**
	 * plan type
	 */
	PlanType string `json:"planType"`
}

File: bitget-node-sdk-api/src/lib/model/mix/plan/ModifyTPSLPlanReq.ts
export class ModifyTPSLPlanReq{
    /**
     * Currency pair
     */
    private _symbol!:string;
    /**
     * Deposit currency
     */
    private _marginCoin!:string;
    /**
     * Order id
     */
    private _orderId!:string;
    /**
     * Trigger price
     */
    private _triggerPrice!:string;
    /**
     * Plan type
     */
    private _planType!:string;


    get symbol(): string {
        return this._symbol;
    }

    set symbol(value: string) {
        this._symbol = value;
    }

    get marginCoin(): string {
        return this._marginCoin;
    }

    set marginCoin(value: string) {
        this._marginCoin = value;
    }

    get orderId(): string {
        return this._orderId;
    }

    set orderId(value: string) {
        this._orderId = value;
    }

    get triggerPrice(): string {
        return this._triggerPrice;
    }

    set triggerPrice(value: string) {
        this._triggerPrice = value;
    }

    get planType(): string {
        return this._planType;
    }

    set planType(value: string) {
        this._planType = value;
    }
}